### PR TITLE
added separate doc page for conform functions and testing

### PR DESCRIPTION
**Pull request by trescube**

### CODE REVIEW

**Comment by trescube** [comment]
@thatdatabaseguy could you check my wording on [Netherlands house numbers](https://github.com/openaddresses/openaddresses/pull/2606/files#diff-2271880c217531c3c2ad6048c2c56dc1R71)?  I'm sure that it's butchered pretty badly.

**Comment by nvkelso** [comment]
This is really good!

File: CONFORM_FUNCTIONS.MD [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+"number": {
+	"function": "format",
+	"fields": ["huisnummer", "huisletter", "huisnummertoevoeging"],
+	"format": "$1$2-$3"
+}
+```
+
+The positional numeric `$` format refers to the 0-based nth element in the `fields` array.  The `-` is a literal hyphen.  
+
+House numbers in the Netherlands are composed of a number concatenated with some optional letters and then a hyphen concatenated optionally with some more numbers.  Some examples of this are:
```
Comment (thatdatabaseguy) [inline CONFORM_FUNCTIONS.MD]: I would say: "House numbers in the Netherlands are composed of a number, an optional letter, and an optional number suffix, which are concatenated using a hyphen. Some examples of this are:"

**Comment by iandees** [comment]
Can the file be renamed so the `.md` is lowercase so as to be consistent with other docs?

**Comment by trescube** [comment]
Renamed (weird commits are due to OSX thinking `.MD` and `.md` are the same thing)

**Pull request merged**